doc/tutorials/space_shooter/step_3.md

```diff
@@ -0,0 +1,134 @@
+just a black screen.
+
+In this step we will look how to improve that, we will replace the static graphic of the player
+to an animation and will create a cool sense of depth and movement by adding a parallax to the
+background of the game.
+
+So lets start by adding the animation to the player ship! For that, we will something that we
+call Sprite Animations, which is an animation that is composed by a collection of sprites, each
+one representing one frame, and the animation effect is achieved by rendering one sprite after
+the other through a time frame.
+
+To better visualize, this is the animation that we will be using, note how this image holds 4
+individual images (or frames).
+
+![](app/assets/images/player.png)
+
+Flame provides us specialized classes to deal with such image: `SpriteAnimation` and its component
+wrapper `SpriteAnimationComponent` and changing our `Player` component to be an animation is quite
+simple, take a look on how the component will look like now:
+
+```dart
+class Player extends SpriteAnimationComponent
+    with HasGameReference<SpaceShooterGame> {
+  @override
+  Future<void> onLoad() async {
+    await super.onLoad();
+
+    animation = await game.loadSpriteAnimation(
+      'player.png',
+      SpriteAnimationData.sequenced(
+        amount: 4,
+        stepTime: .2,
+        textureSize: Vector2(32, 48),
+      ),
+    );
+
+    position = game.size / 2;
+    width = 100;
+    height = 150;
+    anchor = Anchor.center;
+  }
+
+  // Other methods omitted
+}
+```
+
+So lets break down the changes:
+
+- First we changed our `Player` component to extend from `SpriteAnimationComponent` instead of
+`SpriteComponent`
+- In the `onLoad` method we are now using the `game.loadSpriteAnimation` helper instead of the
+ `loadSprite` one, and setting the `animation` attribute with its returned value.
+
+The `SpriteAnimationData` class might look complicated at first glance, but it is actually quite
+simple, note how we used the `sequenced` constructor, which is a helper to load animation images
+where the frames are already layed down in the sequence that they will play, then:
+
+- `amount` defines how many frames that animation have, in this case `4`
+- `stepTime` is the time in seconds that each frame will be rendered, before it gets replaced
+to the next one.
+- `textureSize` is the size in pixels which defines each frame of the image.
+
+With all of these information, the `SpriteAnimationComponent` will now automatically play the
+animation!
+
+Now lets add some depth and energy to our game background. Of course there are many ways of
+doing so, in this tutorial we will explore the idea of the parallax scrolling. If you never heard
+about it, it consist in a technique where background images move past the camera with different
+speeds, this not only creates the sensation of depth but also improve a lot the movement feeling
+of the game. If you want to read more about Parallax Scrolling, check this article
```

Comment:
"but also improves the movement feeling of the game a lot."

Couldn't add multiline suggestion on the phone